borsh-derive/src/lib.rs

```diff
@@ -1,14 +1,24 @@
+#![recursion_limit = "128"]
+#![cfg_attr(
+    feature = "force_exhaustive_checks",
+    feature(non_exhaustive_omitted_patterns_lint)
+)]
+
 extern crate proc_macro;
 use proc_macro::TokenStream;
 use proc_macro2::Span;
 use proc_macro_crate::crate_name;
 use proc_macro_crate::FoundCrate;
 use syn::{parse_macro_input, DeriveInput, Ident, ItemEnum, ItemStruct, ItemUnion};
 
-use borsh_derive_internal::*;
-#[cfg(feature = "schema")]
-use borsh_schema_derive_internal::*;
+mod internals;
```

Comment:
the modules' structure differs by one file from [comment](https://github.com/near/borsh-rs/pull/185#issuecomment-1660867515)
`src/internals/attributes/field.rs` => `src/internals/attributes/field/mod.rs`